configuration/security.md

```diff
@@ -33,14 +33,10 @@ You can, of course, make your Raspberry Pi even more secure by also changing you
 To add a new user with the same permissions as the `pi` user:
 
 ```
-sudo useradd -m fred -G sudo
+sudo adduser fred && sudo adduser fred sudo
 ```
```

Comment:
Should that second part read addgrp?

Reply:
TBH, I have no idea how to use addgrp in combination with adduser.

'sudo adduser fred && sudo adduser fred sudo' works fine. It creates a new user and adds it to the sudo group.

I tried 'sudo adduser fred && sudo addgrp sudo', it creates a new user but returns 'sudo: addgrp: command not found'.

'sudo adduser fred2 addgrp sudo' returns 'adduser: Only one or two names allowed.'

Reply:
From https://manpages.debian.org/jessie/adduser/adduser.8.en.html: 
In synopsis:
'adduser [options] user group'

Further in the document:
'Add an existing user to an existing group
If called with two non-option arguments, adduser will add an existing user to an existing group.'

Reply:
I'm just testing this - will push it soon!